src/applications/claims-status/tests/components/claim-files-tab/AdditionalEvidencePage.unit.spec.jsx

```diff
@@ -192,145 +192,9 @@ describe('<AdditionalEvidencePage>', () => {
       tree
         .getMountedInstance()
         .UNSAFE_componentWillReceiveProps({ uploadComplete: true });
-      expect(getClaimEVSS.calledWith(1)).to.be.true;
+      expect(getClaim.calledWith(1)).to.be.true;
       expect(router.push.calledWith('your-claims/1/files')).to.be.true;
     });
-
-    // START lighthouse_migration
-    context('cst_use_lighthouse feature toggle', () => {
-      const props = {
-        claim,
-        files: [],
-        params,
-        resetUploads: () => {},
-        router: getRouter(),
-        uploadField: { value: null, dirty: false },
-        filesNeeded: [],
-        filesOptional: [],
-      };
-
-      it('calls getClaimLighthouse when enabled, doesnt show va-alerts', () => {
-        // Reset sinon spies / set up props
-        props.getClaimEVSS = sinon.spy();
-        props.getClaimLighthouse = sinon.spy();
-        props.useLighthouse = true;
-        props.filesNeeded = [];
-        props.filesOptional = [];
-
-        const { rerender, container } = render(
-          <AdditionalEvidencePage {...props} />,
-        );
-
-        // We want to trigger the 'UNSAFE_componentWillReceiveProps' method
-        // which requires rerendering
-        rerender(<AdditionalEvidencePage {...props} uploadComplete />);
-
-        expect(props.getClaimEVSS.called).to.be.false;
-        expect(props.getClaimLighthouse.called).to.be.true;
-        expect($('.primary-alert', container)).not.to.exist;
-        expect($('.optional-alert', container)).not.to.exist;
-      });
-
-      it('calls getClaimLighthouse when enabled, shows va-alerts', () => {
```

Comment:
should we keep this test since its a light house test and just rename it or does another test cover this? @jerekshoe